src/App.js

```diff
@@ -6,20 +6,34 @@ import Categories from './components/Categories';
 import './App.css';
 
 function App() {
+  const addActiveClass = (e) => {
+    const navLinks = document.querySelectorAll('nav ul li a');
+    for (let i = 0; i < navLinks.length; i += 1) {
+      navLinks[i].classList.remove('active');
+    }
+    if (e.target === document.querySelector('nav h1')) {
+      document.querySelector('nav ul li a').classList.add('active');
+    } else {
+      e.target.classList.add('active');
+    }
+  };
+
   return (
     <Router>
       <div>
         <nav>
```

Comment:
- [ ] **[OPTIONAL]** Currently, your Navbar is missing the background color in the template.

  I would suggest adding a className to this `<nav>` and then in your **App.css** give it a `background-color` property with the appropriate color (like **#fff**).

  ![shah-nav](https://user-images.githubusercontent.com/36542986/145469302-ffdfa648-195d-4bfd-bbb3-65b2183e5d6b.jpg)